internal/hmyapi/transactionpool.go

```diff
@@ -55,6 +55,20 @@ func (s *PublicTransactionPoolAPI) GetTransactionsHistory(ctx context.Context, a
 	return ReturnWithPagination(result, args), nil
 }
 
+// GetTransactionsHistoryFull returns the list of transactions with full info that involve a particular address.
+func (s *PublicTransactionPoolAPI) GetTransactionsHistoryFull(ctx context.Context, args TxHistoryArgs) ([]*RPCTransaction, error) {
+	hashes, err := s.GetTransactionsHistory(ctx, args)
+	if err != nil {
+		return nil, err
+	}
+	result := []*RPCTransaction{}
+	for _, hash := range hashes {
+		tx := s.GetTransactionByHash(ctx, hash)
```

Comment:
How many hashes could this be? How much on average ? Why aren’t you getting them all at once instead of in sequence ? Does it have to be in sequence ?

Reply:
Pagination makes it to 100 by default, can be more if offset is specified in TxHistoryArgs.